packages/ag-charts-community/src/scene/pattern/pattern.ts

```diff
@@ -34,14 +36,23 @@ export class Pattern implements Omit<RequiredInternalAgPatternColor, 'type'> {
         this.padding = patternOptions.padding ?? 1;
         this.pattern = patternOptions.pattern ?? 'forward-slanted-lines';
         this.rotation = patternOptions.rotation ?? 0;
+        this.scale = patternOptions.scale ?? 1;
         this.path = patternOptions.path;
     }
 
     private getPath(pixelRatio: number) {
         const { pattern, width, height, padding, strokeWidth, path: svgPath } = this;
 
         const path = new ExtendedPath2D();
-        PATTERNS[pattern](path, { width, height, pixelRatio, strokeWidth, padding, svgPath });
+
+        const svgParts = parseSvg(svgPath);
```

Comment:
Just wondering if we should be doing this as an actual validation, which can fail much earlier than here, and give users more specific feedback (e.g. if they are using multiple SVGs)?